### ISSUE TITLE
It cannot be used in phones 

### ISSUE THREAD

**Issue by a1576799** [divamgupta/obsidian-spreadsheets#11]
Could you please make it be used in phones?  Any future plan? Thanks

**Comment by freekvh** [comment]
Seconded. I really need this to sync to the phone. Perhaps it is also good to add to the readme that it does not work on phones because it took me quite some time before I realized that could be the issue :)

Other than this, great plugin.

**Comment by ctwhome** [comment]
If it could be use on the phone to read at least would be fantastic to be able to quickly see the notes. Eventually even better if also could edit.

**Comment by freekvh** [comment]
Yeah Read only would already be great indeed!

**Comment by Fleker** [comment]
Yeah I don't know what limits this plugin from working on a phone but I was surprised it doesn't load at all. Being able to manage a spreadsheet across my phone/laptop is a feature that I'm looking for.
